pkg/debug/scripts/collect-info.sh

```diff
@@ -44,9 +44,9 @@ DIR="$TMP_DIR/$INFO_DIR"
 mkdir -p "$DIR"
 mkdir -p "$DIR/network"
 
-# Install GNU version of the 'ps', 'tar' and other tools
+# Install GNU version of the 'ps' and other tools
```

Comment:
Maybe you could just use: `Install GNU version of the 'ps' and the dmidecode tool`

Reply:
done.